app/src/main/java/com/sdp/movemeet/view/map/MainMapFragment.java

```diff
@@ -178,7 +178,7 @@ else if (!marker.getTag().equals("my position")) {
             } else {
                 intent = new Intent(supportMapFragment.getActivity(), ActivityDescriptionActivity.class);
             }
-            intent.putExtra("activity", act);
+            intent.putExtra(ActivityDescriptionActivity.DESCRIPTION_ACTIVITY_KEY, act);
```

Comment:
Ok nice, you did in fact modified this hard-coded String in the MainMapFragment, well done 👍. Based on this, I also modified the intents in `ProfileActivity` and `EditProfileActivity` accordingly!